app/src/org/commcare/android/tasks/templates/CommCareTask.java

```diff
@@ -30,8 +32,12 @@ protected final C doInBackground(A... params) {
         try {
             return doTaskBackground(params);
         } catch(Exception e) {
+            Logger.log(TAG, e.getMessage());
```

Comment:
Log task crashes.
Also, wrapping an entire method in a generic 'Exception' catch is poor form, so we should fix that one at some point.